test/controllers/api/validate_token_test.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './test_base'
+
+class API::PingControllerTest < API::TestBase
```

Comment:
D'oh. I didn't notice this but the class name should probably be `API::ValidateTokenControllerTest`